arch/x86/sysctx.c

```diff
@@ -0,0 +1,54 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+#include <uk/assert.h>
+#include <uk/arch/ctx.h>
+
+__uptr ukarch_sysctx_get_tlsp(struct ukarch_sysctx *sysctx)
```

Comment:
No reason, I just tend to rely on the fact that LTO would inline these if needed.

Reply:
Actually, another reason would be the fact that I want them to keep the `|isr` attribute. At the moment we have non-ISR code that runs in exception handlers. This is fine if this is included in already ISR files, but If a non ISR function in a non ISR file were to call this then for sure it would not keep the ISR attribute.